### PR TITLE
[ticket/16978] add missing ul closeing tag & convert to twig

### PR DESCRIPTION
**Pull request by marc1706**
master version of https://github.com/phpbb/phpbb/pull/6378

Checklist:

- [x] Correct branch: master for new features; 3.3.x for fixes
- [x] Tests pass
- [x] Code follows coding guidelines: [master](https://area51.phpbb.com/docs/master/coding-guidelines.html) and [3.3.x](https://area51.phpbb.com/docs/dev/3.3.x/development/coding_guidelines.html)
- [x] Commit follows commit message [format](https://area51.phpbb.com/docs/dev/3.3.x/development/git.html)

Tracker ticket (set the ticket ID to **your ticket ID**):

https://tracker.phpbb.com/browse/PHPBB3-16978

### CODE REVIEW

File: phpBB/styles/prosilver/template/posting_pm_header.html [hunk 1]
```diff
@@ -1,83 +1,110 @@
...
+					{% if .to_recipient %}
+						<dd class="recipients">
+							<ul class="recipients">
+								{% for to_recipient in .to_recipient %}
+									<li>
+										{% if to_recipient.IS_GROUP %}
+											<a href="{{ to_recipient.U_VIEW }}"><strong>{{ to_recipient.NAME }}</strong></a>
+										{% else %}
+											{{ to_recipient.NAME_FULL }}
+										{% endif %}&nbsp;
```
Comment (hanakin) [inline phpBB/styles/prosilver/template/posting_pm_header.html:97]: What is the space doing here exactly? And would it not make more sense in the if below?

Comment (marc1706) [reply phpBB/styles/prosilver/template/posting_pm_header.html:97]: Might be a merge conflict resolution issue, will check it

Comment (marc1706) [reply phpBB/styles/prosilver/template/posting_pm_header.html:97]: Exists in the previous code: https://github.com/phpbb/phpbb/blob/master/phpBB/styles/prosilver/template/posting_pm_header.html#L72

Comment (marc1706) [reply phpBB/styles/prosilver/template/posting_pm_header.html:97]: But I agree, makes more sense in the if below.

**Review by hanakin — COMMENTED** [review]
Everything looks good to me but not home at the moment so can not test it at the moment

**Pull request merged**